snippets/2_tricky_strings.py

```diff
@@ -0,0 +1,31 @@
+# 1
+assert id("some_string") == id("some" + "_" + "string")
+assert id("some_string") == id("some_string")
```

Comment:
hmm, when we embed snippets in Readme, we still need to show outputs correct? How'd that work @nifadyev